src/jit/codegenlegacy.cpp

```diff
@@ -11364,8 +11364,6 @@ void CodeGen::genCodeForTreeSmpOpAsg(GenTreePtr tree)
 
             if (VarSetOps::IsMember(compiler, genUpdateLiveSetForward(op2), varDsc->lvVarIndex))
             {
-                noway_assert(compiler->opts.compDbgCode);
```

Comment:
Do you understand why this assert is hitting now, but was (presumably) not hitting before? Is it a new assert, or has something else changed?